### PR TITLE
APT add-on installs also in container-based builds

### PR DESCRIPTION
**Pull request by krlmlr**
See e.g. https://travis-ci.org/rstats-db/RMySQL/builds/119360508#L426

Not sure if I got the links right. Perhaps this information should be placed more prominently.

CC @jimhester @hadley @craigcitro

### CODE REVIEW

**Comment by hadley** [comment]
LGTM, although I think we could add even more about the apt-addon, include an small example, e.g.

```yaml
addons:
  apt:
    packages:
      - libxml2-dev
```

We also need to mention that these dependencies are transitive - i.e. if your package depends on xml2, you'll also need to install `libxml2-dev`

**Comment by krlmlr** [comment]
Could `apt_packages` eventually forward to the `apt-addon`?

**Comment by jimhester** [comment]
@krlmlr Maybe they could be forwarded but not all packages are in the whitelist for the addon so it may result in confusing error messages.

**Comment by krlmlr** [comment]
@jimhester: c55d54d creates an own section for this information, let me know if you prefer the previous version.

File: user/languages/r.md [hunk 1]
```diff
@@ -115,6 +115,32 @@ language: r
 pandoc_version: 1.16
 ```
 
+### APT packages
+
+Use the [APT addon][apt-addon] (currently in beta)
```
Comment (plaindocs) [inline user/languages/r.md]: I *think* we're out of beta?

**Comment by plaindocs** [comment]
Thanks!

**Pull request merged**